src/analytics/PivotTable.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import PivotTableUI from 'react-pivottable/PivotTableUI';
+import 'react-pivottable/pivottable.css';
+
+import { Col } from 'reactstrap';
+
+const PivotTable = ({ cards }) => {
+  const data = cards.map((card) => ({
+    CMC: card.cmc || card.details.cmc,
```

Comment:
This needs to be `card.cmc ?? card.details.cmc` otherwise overrides of 0 don't work.